docs/AppBuildAndDeploy.md

```diff
@@ -21,9 +21,12 @@ stage: Application_CI
     job: Initialise
       steps: 
         task: GetAppVersion
+    job: Lint
```

Comment:
Separated the task from existing build job so that it can run in parallel. Otherwise the build helm chart job has to wait for the build job to complete.